pkg/controller/configmap/configmap_controller.go

```diff
@@ -256,7 +256,7 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 	}
 
 	// Render configurations
-	objs, err := Render(instance, ncpReplicas, opNsxSecret, opLbSecret)
+	objs, err, renderData := Render(instance, ncpReplicas, opNsxSecret, opLbSecret)
```

Comment:
I think it's better to either passing `&ncpReplicas` to `Render`, or reseting `ncpReplicas` in a separate function.

Reply:
Code is modified and in testing.